v2/client_test.go

```diff
@@ -444,10 +448,15 @@ func TestSubscribeNoLeader(t *testing.T) {
 
 func TestSubscribeNotLeaderRetry(t *testing.T) {
 	server := newMockServer()
+	// Set autoClearError for testing retry requests
+	server.mu.Lock()
+	server.autoClearError = true
+	server.mu.Unlock()
```

Comment:
Maybe just make this a function on the mock server `SetAutoClearError`?

Reply:
done